Tasks/AzureResourceGroupDeployment/operations/Utils.ts

```diff
@@ -10,6 +12,17 @@ class Utils {
         }
         return error;
     }
+
+    public static async getTargetUriFromFwdLink(fwdLink: string) {
+        tl.debug("Trying to fetch target link from the fwdlink: " + fwdLink);
+        var httpRequest = new webClient.WebRequest();
+        httpRequest.method = 'GET';
+        httpRequest.uri = fwdLink;
```

Comment:
Same question as in the other #6484 , do you have to set request.AllowAutoRedirect = false explicitly?

Reply:
for this version of typed-rest-client, the httpclient does not handle auto-redirect of get requests. Those changes were added later. So we should be good here. the changes made to support autoredirect allow redirecting by default, and so it will cause a problem If someone bumps up the version of typed-rest-client. I will add an Integration Test for catching the same.